uxarray/grid/geometry.py

```diff
@@ -1407,25 +1466,14 @@ def _populate_bounds(
     """
 
     # Ensure grid's cartesian coordinates are normalized
+    # TODO: Is it possible to have a more flexible normalization? (Avoid duplicated normalizations)
```

Comment:
@ philipc2 Can we avoid duplicate normalization here? For example if the nodes are already normalized, we will not normalize them.

For example, set an attributes in the grid `node_x`,`node_y`,`node_z` to indicate if they're already normalized or not.

Reply:
I believe that we handle this here

https://github.com/UXARRAY/uxarray/blob/a6aa6294975f189e8c36dd3d14da745526e53e06/uxarray/grid/grid.py#L1983